scene/multiplayer/scene_replication_interface.cpp

```diff
@@ -309,6 +309,7 @@ Error SceneReplicationInterface::on_despawn_receive(int p_from, const uint8_t *p
 	Error err = rep_state->peer_del_remote(p_from, net_id, &node);
 	ERR_FAIL_COND_V(err != OK, err);
 	ERR_FAIL_COND_V(!node, ERR_BUG);
+	node->get_parent()->remove_child(node);
```

Comment:
Better to also check `node->get_parent() != nullptr`, very unlikely but at least we'll be able to better catch bugs if they happen.